readthedocs/search/tests/conftest.py

```diff
@@ -20,7 +20,7 @@ def es_index():
 
 
 @pytest.fixture(autouse=True)
-def all_projects(es_index, mock_processed_json):
+def all_projects(es_index, mock_processed_json, db):
```

Comment:
This doesn't appear to be used. Is it necessary?

Reply:
Its actually a fixture which need to injected for db access. whether you use or not, if any fixture has dependency to `db` fixture, that means the fixture has access to database.